### ISSUE TITLE
Not clear installation instruction

### ISSUE THREAD

**Issue by eugenberend** [kubernetes-sigs/kubectl-validate#151]
### What happened?

➜  ~ go install sigs.k8s.io/kubectl-validate@latest

<...multiple lines of installation log messages here ...>

➜  ~ kubectl validate
error: unknown command "validate" for "kubectl"

What I should do to use this command?

### What did you expect to happen?

kubectl validate works, obviously

### How can we reproduce it (as minimally and precisely as possible)?

follow your own installation instructions

### Kubernetes version

<details>

```console
$ kubectl version
Client Version: v1.31.3
```

</details>

**Comment by jepfred** [comment]
I had the same experience. I'm on macOS 15.1.1 (24B91). I did find the binary in `~/go/bin` and could invoke it, and it works, but `kubectl validate` simply returns `error: unknown command "validate" for "kubectl"`.

**Comment by nicl-dev** [comment]
I'm on macOS 15.2 with the same problem. Invoking the binary works, `kubectl validate` returns `error: unknown command "validate" for "kubectl"`,

**Comment by miraccan00** [comment]
# Installing and Using `kubectl-validate` on Windows  

On my Windows system, I was able to install the `kubectl-validate` tool without any issues using the following command:  

```sh
go install sigs.k8s.io/kubectl-validate@latest
```

After installation, I successfully used the `kubectl-validate` command to validate a Kubernetes YAML file:  

```sh
kubectl-validate deployment.yaml
```

### Output:  
```
deployment.yaml...OK
```

This confirms that the file was validated successfully.  

Additionally, when I checked my `kubectl` version, I got the following output:  

```sh
kubectl version
```

### Output:  
```
Client Version: v1.31.4
```

So, both the `kubectl-validate` tool and my `kubectl` client are working correctly.  

## Additional Note for macOS  
For macOS users, they might need to add the binary path to the OS environment variables to use `kubectl-validate` properly.

**Comment by tedivm** [comment]
Yeah it would be nice if the instructions for installation actually worked.
